wagtail/contrib/settings/permissions.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.contenttypes.models import ContentType
+
+
+def user_can_edit_setting_type(user, model):
+    """ Check if a user has any permission related to a content type """
```

Comment:
The practice of saying "allow users to access this area if they have *any* permission over model X" is something that we've tightened up a lot in Wagtail 1.1 - we now tend to list the specific permissions, e.g. https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailredirects/views.py#L17. (Amongst other things, this opens up the possibility of using permissions for 'lesser' tasks that don't require access to the model's admin area - "can download document", "can link to image" and so on.)

In the case of settings, it probably makes sense for `change_[foo]` to be the only permission type in use - so that would be the one that gets checked in the view and `MenuItem.is_shown`, and the one that gets passed to `register_permissions` to appear in the groups admin (if we go ahead with that).